cilium-cli/clustermesh/clustermesh_test.go

```diff
@@ -711,5 +650,147 @@ func TestGetCASecret(t *testing.T) {
 			tt.assertErr(t, err)
 		})
 	}
+}
+
+func TestGetClustersFromValues(t *testing.T) {
+	uu := map[string]struct {
```

Comment:
nit: I would make it a list with field "name", similar to `TestGetCASecret`. 
It will be a bit easier to ready with one less indent.